onfido-java/src/main/models/location.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Location",
+  "properties": {
+    "ip_address": {
+      "type": "string",
+      "description": "The applicant's ip address"
+    },
+    "country_of_residence": {
+      "type": "string",
+      "description": "The applicant's country of residence"
+    }
+  }
+}
```

Comment:
Missing new line

Reply:
Added new line at the end of the file, it should be good now.